packages/plugin-lighthouse/src/lib/utils.ts

```diff
@@ -68,3 +75,89 @@ export function validateOnlyAudits(
   }
   return true;
 }
+
+type UnsupportedDetail =
+  | Details.CriticalRequestChain
+  | Details.List
+  | Details.TreemapData
+  | Details.Screenshot
+  | Details.Filmstrip
+  | Details.DebugData;
+type UnsupportedDetailTypes = UnsupportedDetail['type'];
+
+export function toAuditOutputs(
+  lhrAudits: Result[],
+): AuditOutputs {
+  return Object.values(lhrAudits).map(
```

Comment:
It's already an array:

```suggestion
  return lhrAudits.map(
```